File: Modul 4/Source Code/Python - Modul 4/PRAK404-2410817310008-MAnshary.py
while True:
    print("Pilih program: ")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    pilihan = int(input("Masukkan Pilihan: "))

    if pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator M.Anshary")
        break
    elif pilihan in {1, 2, 3, 4}:
        a = float(input("Masukkan nilai pertama: "))
        b = float(input("Masukkan nilai kedua: "))
        
        if pilihan == 1:
            print(f"Hasil Penjumlahan antara {a:.2f} dan {b:.2f} adalah {a + b:.2f}")
        elif pilihan == 2:
            print(f"Hasil Pengurangan antara {a:.2f} dan {b:.2f} adalah {a - b:.2f}")
        elif pilihan == 3:
            print(f"Hasil Perkalian antara {a:.2f} dan {b:.2f} adalah {a * b:.2f}")
        elif pilihan == 4:
            if b != 0:
                print(f"Hasil Pembagian antara {a:.2f} dan {b:.2f} adalah {a / b:.2f}")
            else :
                print("Error: Pembagian dengan Nol.")
    else:
        print("Input anda salah, silahkan coba lagi")

File: Modul 4/Source Code/C - Modul 4/PRAK404-2410817310008-MAnshary.c
#include<stdio.h>
int main(){
    int pilihan;
    float a, b, hasil; 

    while(pilihan != 5){
        printf("Pilih Program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &pilihan);

        if(pilihan==5){
            printf("Terimakasih telah menggunakan Kalkulator M.Anshary");
            break;
        }
        else if(pilihan>=1 && pilihan<=4){
            printf("Masukkan Nilai Pertama: ");
            scanf("%f", &a);
            printf("Masukkan Nilai Kedua: ");
            scanf("%f", &b);

            if(pilihan==1){
                hasil=a+b;
                printf("Hasil Penjumlahan antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            }
            else if(pilihan==2){
                hasil=a-b;
                printf("Hasil Pengurangan antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            }
            else if(pilihan==3){
                hasil=a*b;
                printf("Hasil Perkalian antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
            }
            else{
                if(b!=0){
                    hasil=a/b;
                    printf("Hasil Pembagian antara %.2f dan %.2f adalah %.2f\n", a, b, hasil);
                }
                else{
                    printf("Error: Pembagian dengan Nol.\n");
                }
            }
        }
        else{
            printf("Input anda salah, silahkan coba lagi.\n");
        }
    }
    return 0;
}